crates/sui/src/keytool.rs

```diff
@@ -155,18 +155,24 @@ impl KeyToolCommand {
             }
 
             KeyToolCommand::LoadKeypair { file } => {
-                let res: Result<SuiKeyPair, anyhow::Error> = read_keypair_from_file(&file);
-
-                match res {
+                match read_keypair_from_file(&file) {
                     Ok(keypair) => {
+                        // Account keypair is encoded with the key scheme flag {},
```

Comment:
Do we prefer not storing the flag for anything apart from account keys? Atm it makes sense as we don't provide crypto agility in these operations, we might however want to reconsider this in the future to have some consistency re key management.

Reply:
agreed, left it because this involves adding BLS to enum, therefore to all switch clause on key types. it is quite a large change that i decide to leave out, but worth a revisit to add them eventually.